### DISCUSSION TITLE
关于 ssmanager 的一些疑问

### DISCUSSION THREAD

**Discussion by pexcn** [shadowsocks/shadowsocks-rust#493] [Q&A] [answered]
它和 ssserver 相比缺少 `--udp-timeout` 和 `--udp-max-associations` 选项，能加上吗？

**Comment by zonyitoo** [comment, marked as answer]
You can set it in configuration, BTW.

**Reply by pexcn** [reply]
在 docker 里运行，本来是纯命令的，设置这两个参数要映射一个文件，多维护一份配置，感觉这样不太方便...
而且 ssserver 和 sslocal 都是支持这两个参数的，能同步支持一下吗😄

**Reply by zonyitoo** [reply]
Sounds reasonable. Please open an issue about it.

**Reply by pexcn** [reply]
> Sounds reasonable. Please open an issue about it.

Move: https://github.com/shadowsocks/shadowsocks-rust/issues/494
